pennylane/beta/plugins/default_tensor.py

```diff
@@ -80,7 +80,9 @@ class DefaultTensor(Device):
       >>> mps_tensornet = qml.device("default.tensor", wires=2, representation="mps")
```

Comment:
This device needed to be refactored now that ``Device`` passes ``Wires`` in the ``execute`` function to ``apply``.